packages/custom_lint/test/src/workspace_test.dart

```diff
@@ -90,11 +90,20 @@ Future<Directory> createProject(
   Directory dir,
   Pubspec pubspec, {
   required List<Map<String, Object?>> packageConfigs,
+  bool createNestedAnalysis = false,
```

Comment:
I don't think that's the responsibility of this function.

The tests should do this instead